.gitignore

```diff
@@ -1,3 +1,4 @@
 /composer.lock
 /tests/phpunit.xml
 /vendor
+.idea
```

Comment:
You should add this into your global gitignore, not here: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore